### PR TITLE
[IMP] stock_return_request: lot suggestions

### PR DESCRIPTION
**Pull request by chienandalu**
Lot suggestion wizard so the user can easily guess the lot that is going to be returned.

image

cc @Tecnativa TT20569

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Travis is failing

File: stock_return_request/wizard/suggest_return_request_lot.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        if (
+            self.env.context.get("active_model", False)
+            != "stock.return.request.line"
+        ):
+            return (False, False)
+        request_line = (
+            self.request_line_id or
+            self.request_line_id.browse(self.env.context.get('active_id')))
+        if not request_line:
+            (False, False)
```
Comment (carlosdauden) [inline stock_return_request/wizard/suggest_return_request_lot.py]:
```suggestion
            return (False, False)
```

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: stock_return_request/wizard/suggest_return_request_lot.py (hunk 1 above)
Comment (chienandalu) [reply stock_return_request/wizard/suggest_return_request_lot.py]: Sure :) thanks

**Review by carlosdauden — APPROVED** [review]

File: stock_return_request/wizard/suggest_return_request_lot.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            return
+        return [(
+            ml.lot_id.id, '{} - {} - {} - {}'.format(
+                ml.date, ml.lot_id.name,
+                ml.reference, suggested_lots_moves[ml])
+        ) for ml in suggested_lots_moves.keys()]
+
+    def action_confirm(self):
+        if self.lot_suggestion_mode == 'sum' and self.suggested_lot:
+            self.request_line_id.lot_id = self.suggested_lot
```
Comment (pedrobaeza) [inline stock_return_request/wizard/suggest_return_request_lot.py:96]: I don't get how this is working assigning that selection value. This seems very hackish...

Comment (chienandalu) [reply stock_return_request/wizard/suggest_return_request_lot.py:96]: Well, the selection value is the id of the lot. Not black magic was made :)

Comment (pedrobaeza) [reply stock_return_request/wizard/suggest_return_request_lot.py:96]: Ah, OK, I see. Then add a mention that on next version, you can't put an integer as selection key (only strings), so you have to convert from/to in this code.

Comment (chienandalu) [reply stock_return_request/wizard/suggest_return_request_lot.py:96]: Added comment

**Pull request merged**